examples/cloudflare-modules/src/index.ts

```diff
@@ -1,22 +1,27 @@
 // src/index.mjs
 import { createYoga, createSchema } from 'graphql-yoga'
 
-const { fetch } = createYoga({
-  graphqlEndpoint: '/graphql',
-  landingPage: false,
-  schema: createSchema({
-    typeDefs: /* GraphQL */ `
-      type Query {
-        greetings: String
-      }
-    `,
-    resolvers: {
-      Query: {
-        greetings: () =>
-          'This is the `greetings` field of the root `Query` type',
-      },
+const schema = createSchema({
+  typeDefs: /* GraphQL */ `
+    type Query {
+      greetings: String
+    }
+  `,
+  resolvers: {
+    Query: {
+      greetings: () => 'This is the `greetings` field of the root `Query` type',
     },
-  }),
+  },
 })
 
-export default { fetch }
+export default {
+  fetch(request: Request, env: { [key: string]: string }) {
+    const yoga = createYoga({
```

Comment:
Maybe remove the parameter and modify the bundled code inside the deployment script;
https://github.com/dotansimha/graphql-yoga/pull/1812/files
Or do you think it is too tricky?

Reply:
I feel like it's too tricky... see my other comments on why we need this setup

Reply:
But this is also our example for CF Modules and `useLandingPage` and having `yoga` in the fetch function etc are too much for a basic example. We can have another project for e2e tests but we will lose the ability of testing the actual example in this case.

Reply:
I see, so let's move on with this PR as-is. Then, we'll open an issue to improve the setup. We'll need to experiment with using the router config that includes `*`, then we can use `/WORKER_ID/graphql` and keep the landing page enabled.